scripts/apps/archive/controllers/BaseListController.js

```diff
@@ -8,9 +8,10 @@ export class BaseListController {
         $scope.selected = {};
 
         $scope.fetchNext = (from) => {
-            var criteria = this.getQuery();
+            var criteria = this.getQuery(null, $scope.repo.archive || false);
 
             criteria.from = from;
+            criteria.size = 50;
```

Comment:
You are already setting the size to 50 in the other file, so you can remove this line.